website/pages/en/network/developing.mdx

```diff
@@ -2,52 +2,99 @@
 
-Once the Subgraph Developer is ready to update, they can initiate a transaction to point their subgraph at the new version. Updating the subgraph migrates any signal to the new version (assuming the user who applied the signal selected "auto-migrate"), which also incurs a migration tax. This signal migration should prompt Indexers to start indexing the new version of the subgraph, so it should soon become available for querying.
+### Context for Subgraph Development
 
-### Deprecating Subgraphs
+Great subgraphs start with a local development environment and unit tests. Developers use [Graph CLI](https://github.com/graphprotocol/graph-tooling/tree/main/packages/cli), a command-line interface tool for building and deploying subgraphs in The Graph. They can also use [Graph TypeScript](/developing/graph-ts/) and [Matchstick](/developing/unit-testing-framework/) to create robust subgraphs that deploy to The Graph Network.
 
-At some point a developer may decide that they no longer need a published subgraph. At that point they may deprecate the subgraph, which returns any signalled GRT to the Curators.
+## Deploy to Subgraph Studio
 
-### Diverse Developer Roles
+Once defined, a subgraph can be deployed to [Subgraph Studio](/deploying/deploying-a-subgraph-to-studio/). In Subgraph Studio, you can do the following:
 
-Some developers will engage with the full subgraph lifecycle on the network, publishing, querying and iterating on their own subgraphs. Some may be focused on subgraph development, building open APIs which others can build on. Some may be application focused, querying subgraphs deployed by others.
+- Use its staging environment to index the deployed subgraph and make it available for review.
+- Verify that your subgraph doesn't have any indexing errors and works as expected.
 
-### Developers and Network Economics
+## Publish to the Network
 
-Developers are a key economic actor in the network, locking up GRT in order to encourage indexing, and crucially querying subgraphs, which is the network's primary value exchange. Subgraph developers also burn GRT whenever a subgraph is updated.
+When you are happy with your subgraph, you can put it into production and [publish it](/publishing/publishing-a-subgraph/) to The Graph Network. This ensures Indexers can index it.
+
+- This is an on-chain action, which registers the subgraph and makes it discoverable by Indexers.
+- Published subgraphs have a corresponding NFT, which defines the ownership of the subgraph. You can [transfer the subgraph's ownership](/managing/transfer-and-deprecate-a-subgraph) by sending the NFT.
+- Published subgraphs have associated metadata, which provides other network participants with useful context and information.
+
+## Add Signal for Indexing
+
+Published subgraphs are unlikely to be picked up by Indexers without signal. To encourage indexing you should add signal to your subgraph. Learn more about signaling and [curating](/network/curating/) on The Graph.
```

Comment:
To make this a bit more clear, should we consider replacing "signal" with "curation signal"?

Someone less familiar with The Graph might see "signal"and be confused. Whether we use one or the other, we should do so consistently throughout the page (and the rest of the docs).